packages/circuit-ui/package.json

```diff
@@ -99,8 +99,8 @@
     "@emotion/styled": "10.x",
     "@emotion/stylis": "0.8.x",
     "@sumup/collector": "1.x",
-    "@sumup/design-tokens": ">=1.0.0 <3.0.0",
-    "@sumup/icons": "1.x",
+    "@sumup/design-tokens": ">=2.2.0",
+    "@sumup/icons": "1.7.0",
```

Comment:
@robinmetral We need to make sure to undo these changes before merging this PR.

Reply:
@connor-baer done, can you double-check?